### ISSUE TITLE
Improve our CodeFactor mark

### ISSUE THREAD

**Issue by mossroy** [kiwix/kiwix-js#639]
CodeFactor does not give us a very good mark for now : C-
especially for app.js (F) : https://www.codefactor.io/repository/github/kiwix/kiwix-js/file/master/www/js/app.js

There are many minor issues that could easily be fixed.
And we also might configure it to ignore the external libraries.

**Comment by Jaifroid** [comment]
I agree. Codefactor even offers to make PRs for simple fixes. Might be worth experimenting with that, if the proposed fixes are sensible.

**Comment by ram690514** [comment]
@kelson42  i would like to work on this issue

**Comment by kelson42** [comment]
@ram690514 You are welcome.

**Comment by mossroy** [comment]
@ram690514 : I'd prefer that you finalize work on #660 PR first, before starting work on another issue.

**Comment by ram690514** [comment]
@mossroy :I am working on it . I will fix it surely as fast as possible.

**Comment by ram690514** [comment]
@kelson42 : I have pushed a pull request . please check it

**Comment by ram690514** [comment]
@mossroy : I have pushed a pull request. can you please check it

**Comment by Jaifroid** [comment]
@ram690514 Thank you, but it's impossible to see what you have actually fixed from that PR. Could you please close it and re-do it using with only the actual code changes, or perhaps with one commit per change, Please do not change formatting and indenting, only focus on the actual code changes. If the code change isn't clearly shown in the diff, we cannot see what you have done. and so cannot check your work!

**Comment by ram690514** [comment]
@Jaifroid  I will surely do it

**Comment by ram690514** [comment]
I have created a pull request regarding this issue..please check @kelson42

**Comment by ram690514** [comment]
@Jaifroid : I have pushed a pull request.please check it.

**Comment by Jaifroid** [comment]
I've reviewed it and suggested changes. As a matter of etiquette, when you create a PR, please give it an informative title ("Changes" means nothing and could apply to any PR), and also provide an overview description of what you have done. You don't need to provide a description now, as we know what you have done, but it is good practice for a PR. You should change the PR title, however.

**Comment by ram690514** [comment]
Thanks for your your suggestions, I will surely do it

**Comment by ram690514** [comment]
@Jaifroid : I have updated the pull request ,please check it

**Comment by ram690514** [comment]
@kelson42 : I have updated the pull request,please check it

**Comment by ram690514** [comment]
I had restored the element message channel and updated the pull request

**Comment by Jaifroid** [comment]
Well, I was wrong. Merging this went from a score of C- to a score of B !
